controllers/movies.ts

```diff
@@ -0,0 +1,171 @@
+import { error } from "console";
+import { Movie } from "../model/Movie";
+import { ILike } from "typeorm";
+
+export const getAllMovies= async( req:any, res: any, next:any)=>{
+
+    try{
+        
+        const { actor, imdbSort, yearSort  }= req.query;
+        let query= Movie.createQueryBuilder("movie");
+        if(actor){
+            query=query.where("movie.Actors LIKE :actor", {actor:`%${actor}%`});
+        } 
+        if(imdbSort){
+            const imdbSortDirection=imdbSort.toUpperCase()==="ASC" ? "ASC" :"DESC";
```

Comment:
Leave spaces before and after "=". Format your document. Right click on Visual Studio Code and  select Format Document